core/store/orm/orm.go

```diff
@@ -667,6 +667,15 @@ func (orm *ORM) GetConfigValue(field string, value encoding.TextUnmarshaler) err
 	return value.UnmarshalText([]byte(config.Value))
 }
 
+// GetServiceLogLevel returns the log level for a configured service
+func (orm *ORM) GetServiceLogLevel(serviceName string) (string, error) {
```

Comment:
I think we want to go with the jpv2 pattern for orm/model stuff. Which would mean a models.go/orm.go file in a services/logconfig package. Cc @spooktheducks do you agree?

Reply:
Yeah, this will help us disentangle all of the latent import cycles lurking in the codebase

Reply:
updated and separated out to logger/orm & logger/model~